runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBrowseView/Header.tsx

```diff
@@ -27,10 +29,36 @@ interface HeaderProps {
     className?: string;
 }
 
+enum ProcesTabs {
+    PROCESSES = 'processes',
+    COLLECTIONS = 'collections'
+}
+
 const Header: FC<HeaderProps> = ({ className, ...rest }) => {
     const { translate } = useTranslations();
     const hasProcessAddAccess = useFeatureKey([FeatureKey.PROCESS_ADD]);
     const hasAddCollectionAccess = useFeatureKey([FeatureKey.PROCESS_COLLECTION_ADD]);
+    
+    const router = useRouter();
+    const currentTab = router.asPath.split('/').slice(-1)[0].split('?')[0];
+    const { firstValueFrom } = useQuery();
+    const collectionId = firstValueFrom('collection') ?? 1;
+
+    const onTabChange = (event: ChangeEvent<HTMLInputElement>, value: ProcesTabs) => {
+        if (value === ProcesTabs.COLLECTIONS) router.push(`/app/processes/collections?collection=${collectionId}`, null, { locale:router.locale });
```

Comment:
we cannot remove other params responsible for the processes pagination. 

and space in router:locale